tests/ops/qubit/test_parametric_ops.py

```diff
@@ -2905,7 +2905,7 @@ def test_pauli_rot_identity_torch(self, torch_device, theta):
 
         val = np.cos(-theta / 2) + 1j * np.sin(-theta / 2)
         exp = torch.tensor(np.diag([val, val]), device=torch_device)
-        assert torch.allclose(mat, exp)
+        assert qml.math.allclose(mat, exp)
```

Comment:
Was getting "`ComplexDouble` is not equal to `ComplexFloat`" errors with v1.13. `qml.math` ignores the typing.